starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,11 @@
+body {
+background-color:white;
+max-width: 720px; 
+text-align: center;
```

Comment:
To center everything I recommend you this guide about centering in CSS

https://css-tricks.com/centering-css-complete-guide/#center-horizontally-and-vertically